Tinder-Habit/UserHandler.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  UserHandler.swift
+//  Tinder-Habit
+//
+//  Created by Trang Doan on 12/8/20.
+//  Copyright © 2020 Trang Doan. All rights reserved.
+//
+
+import Foundation
+import FirebaseDatabase
+
+//temporary
+var users:[User?] = []
```

Comment:
I think I'd get rid of the `?` here.  If you keep it, you'll have to check each element of the array to make sure there's really a user there and it's not `nil`.  If you just make it `[User]` you can avoid that check.  Then if there's an error or something getting one of the users, just don't put the result in the array.